Chap_API_Job_Mgmt.tex

```diff
@@ -886,48 +886,59 @@ \subsection{\code{PMIx_Log}}
 \returnsimple
 
 \reqattrstart
-If the \ac{PMIx} library does not itself perform this operation, then it is required to pass any attributes provided by the client to the host environment for processing. In addition, it must include the following attributes in the passed \refarg{info} array:
+If the \ac{PMIx} library does not itself perform this operation, then it is required to pass any attributes provided by the client to the host environment for processing. In addition, it must include the following attributes in the passed \refarg{directives} array:
 
 \pasteAttributeItem{PMIX_USERID}
 \pasteAttributeItem{PMIX_GRPID}
 
-Host environments or \ac{PMIx} libraries that implement support for this operation are required to support the following attributes:
+Host environments or \ac{PMIx} libraries that implement support for this operation are required to support the following attributes when provided as part of the \refarg{data} array:
 
 \pasteAttributeItem{PMIX_LOG_STDERR}
 \pasteAttributeItem{PMIX_LOG_STDOUT}
 \pasteAttributeItem{PMIX_LOG_SYSLOG}
-\pasteAttributeItem{PMIX_LOG_LOCAL_SYSLOG}
-\pasteAttributeItem{PMIX_LOG_GLOBAL_SYSLOG}
+
+Similarly, the following attributes must be supported when provided as part of the \refarg{directives} array:
+
 \pasteAttributeItem{PMIX_LOG_SYSLOG_PRI}
 \pasteAttributeItem{PMIX_LOG_ONCE}
 
 \reqattrend
 
 \optattrstart
-The following attributes are optional for host environments or \ac{PMIx} libraries that support this operation:
+The following attributes are optional for host environments or \ac{PMIx} libraries that support this operation when provided as part of the \refarg{data} array:
+
+\pasteAttributeItem{PMIX_LOG_EMAIL}
+\pasteAttributeItem{PMIX_LOG_JOB_RECORD}
+\pasteAttributeItem{PMIX_LOG_GLOBAL_DATASTORE}
+\pasteAttributeItem{PMIX_LOG_LOCAL_SYSLOG}
+\pasteAttributeItem{PMIX_LOG_GLOBAL_SYSLOG}
+
+Similarly, the following attributes are optional when provided as part of the \refarg{directives} array:
 
 \pasteAttributeItem{PMIX_LOG_SOURCE}
 \pasteAttributeItem{PMIX_LOG_TIMESTAMP}
 \pasteAttributeItem{PMIX_LOG_GENERATE_TIMESTAMP}
 \pasteAttributeItem{PMIX_LOG_TAG_OUTPUT}
 \pasteAttributeItem{PMIX_LOG_TIMESTAMP_OUTPUT}
 \pasteAttributeItem{PMIX_LOG_XML_OUTPUT}
-\pasteAttributeItem{PMIX_LOG_EMAIL}
-\pasteAttributeItem{PMIX_LOG_EMAIL_ADDR}
-\pasteAttributeItem{PMIX_LOG_EMAIL_SENDER_ADDR}
```

Comment:
A number of these have been removed from this list but not reintroduced in another list, should they be added to the list of optional attributes above?

Reply:
`PMIX_LOG_EMAIL` is still in the list, just moved to the section immediately above this one. The other `LOG_EMAIL_xxx` attributes are listed in the `PMIX_LOG_EMAIL` description as they are to be included in the `pmix_data_array_t` for that attribute. We can split them into a list within that description (as was done for the stats area) if that helps.